inc/command/class-command.php

```diff
@@ -188,14 +202,224 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( $input, $output, 'vendor/bin/phpunit', $options );
 	}
 
+	/**
+	 * Runs Codecept with zero config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function codecept( InputInterface $input, OutputInterface $output ) {
+		$options = $input->getArgument( 'options' );
+		$module = $input->getOption( 'module' );
+		$run_headless_browser = $input->getOption( 'browser' );
```

Comment:
I'd not force the container to run each time, not all suites will have acceptance tests that would use the web driver. Maybe documentation can help with this ?

Reply:
Yeah it could do. At a lower level maybe if we bundle a helper module it can handle starting / stopping the container.

If we have known suite names that we provision we could check if it's the acceptance suite but lets iterate on that later 👍